charts/incubator/privatebin/Chart.yaml

```diff
@@ -0,0 +1,25 @@
+apiVersion: v2
+appVersion: 1.3.5
+description: PrivateBin running on an Nginx, php-fpm & Alpine Linux stack
+name: privatebin
+version: 4.3.1
+kubeVersion: ">=1.16.0-0"
+keywords:
+- privatebin
+- pastebin
+home: https://github.com/k8s-at-home/charts/tree/master/charts/incubator/privatebin
+icon: https://privatebin.info/theme/img/icon.svg
+sources:
+- https://github.com/PrivateBin/PrivateBin
+- https://github.com/PrivateBin/docker-nginx-fpm-alpine
+maintainers:
+- name: psych0d0g
+  email: psych0d0g@users.noreply.github.com
+dependencies:
+- name: common
+  repository: https://library-charts.k8s-at-home.com
+  version: 4.2.0
+- name: postgresql
+  version: 10.13.10
+  repository: https://charts.bitnami.com/bitnami
+  condition: postgresql.enabled
```

Comment:
Please add an annotation to the chart with the changes. Check out [the template chart](https://github.com/k8s-at-home/charts/blob/master/templates/chart/Chart.yaml) for an example.